assets/src/amp-validation-error-detail-toggle.js

```diff
@@ -54,7 +61,20 @@ function addToggleListener() {
 	} );
 }
 
+function addViewErrorsByTypeLinkButton() {
+	if ( 'undefined' === typeof errorIndexAnchor || 'undefined' === typeof errorIndexLink ) {
+		return;
+	}
+	const heading = document.querySelector( '.wp-heading-inline' );
+	const link = document.createElement( 'a' );
+	link.innerHTML = errorIndexAnchor;
```

Comment:
`innerText` should be used instead of `innerHTML` because the latter is susceptible to an injection attack, if `errorIndexAnchor` is coming from an untrusted source.